### PR TITLE
added gemfile for simple_form

### PR DESCRIPTION
**Pull request by neel7927**
Added simple_form gemfile

### CODE REVIEW

**Comment by ATrillionCoder** [comment]
Nice add!!

**Comment by ipeer28** [comment]
simple_form added

**Comment by Sajjaad1509** [comment]
Looks good

File: Gemfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'simple_form', github: 'heartcombo/simple_form'
```
Comment (ipeer28) [inline Gemfile:24]: indentation good

**Pull request merged**